RELEASE-NOTES.md

```diff
@@ -9,6 +9,9 @@
 * Renamed `SerializerFactory::newSnaksSerializer` to `SerializerFactory::newSnakListSerializer`, leaving a b/c alias
 * Renamed `DeserializerFactory::newClaimDeserializer` to `DeserializerFactory::newStatementDeserializer`, leaving a b/c alias
 * Renamed `DeserializerFactory::newSnaksDeserializer` to `DeserializerFactory::newSnakListDeserializer`, leaving a b/c alias
+* Added `SerializerFactory::newStatementListSerializer`
+* Added `DeserializerFactory::newStatementListDeserializer`
+* Added `DeserializerFactory::newFingerprintDeserializer`
```

Comment:
To be more precise, why is the deserializer public via the factory but the serializer is not?